rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3FilterHandler.java

```diff
@@ -126,4 +154,20 @@ private boolean isAcceptedAddress(final InetAddress address) {
         }
     }
 
+    private boolean isIpAddress(String address) {
+        boolean isIPv4 = false;
+        boolean isIPv6 = false;
+
+        if (address != null) {
+            try {
+                InetAddress inetAddress = InetAddress.getByName(address);
+                isIPv4 = (inetAddress instanceof Inet4Address) && inetAddress.getHostAddress().equals(address);
+                isIPv6 = (inetAddress instanceof Inet6Address);
```

Comment:
Probably I'm missing something, but why we don't need to check `inetAddress.getHostAddress().equals(address)` for ip v6?

Reply:
Done